client-v2/src/components/FrontsEdit/FrontContainer.tsx

```diff
@@ -50,6 +51,11 @@ const FrontsContainer = styled('div')`
   transform: translate3d(0, 0, 0);
 `;
 
+const FrontHeaderButton = styled(Button)`
+  color: ${({ theme }) => theme.shared.colors.greyMediumDark};
+  padding: 0 5px;
```

Comment:
Is there a way I could avoid this with the `Button` properties?

Reply:
@ajwl and I had a brief chat offline -- we could absolutely have a prop on the Button component for button content that required minimal padding, possibly in a future PR 👍